blog

```diff
@@ -0,0 +1,154 @@
+    <img class="adoption-pic" src="images/piper-card.jpg">
+     <h3>Piper</h3>       
+    <p class="dog-cost">Cost to Adopt: $123.45</p> 
+    <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Cras maximus feugiat elementum. Pellentesque aliquet in lacus laoreet hendrerit. Proin eget tellus odio. Sed posuere sem eget blandit cursus. Curabitur venenatis, leo a malesuada sollicitudin, nunc diam luctus mauris, id semper urna massa maximus orci. </p>
+     <button class="adopt-button"><p>Adopt</p></button>
+        </section>
+    <section class="dog-holder">
+    <img class="adoption-pic" src="images/maximus-card.jpg">
+     <h3>Maximus</h3>       
+    <p class="dog-cost">Cost to Adopt: $123.45</p> 
+    <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Cras maximus feugiat elementum. Pellentesque aliquet in lacus laoreet hendrerit. Proin eget tellus odio. Sed posuere sem eget blandit cursus. Curabitur venenatis, leo a malesuada sollicitudin, nunc diam luctus mauris, id semper urna massa maximus orci. </p>
+     <button class="adopt-button"><p>Adopt</p></button>
+        </section>
+    <section class="dog-holder">
+    <img class="adoption-pic" src="images/luna-card.jpg">
+     <h3>Luna</h3>       
+    <p class="dog-cost">Cost to Adopt: $123.45</p> 
+    <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Cras maximus feugiat elementum. Pellentesque aliquet in lacus laoreet hendrerit. Proin eget tellus odio. Sed posuere sem eget blandit cursus. Curabitur venenatis, leo a malesuada sollicitudin, nunc diam luctus mauris, id semper urna massa maximus orci. </p>
+     <button class="adopt-button"><p>Adopt</p></button>
+        </section>
+    <section class="dog-holder">
+    <img class="adoption-pic" src="images/stella-card.jpg">
+     <h3>Stella</h3>       
+    <p class="dog-cost">Cost to Adopt: $123.45</p> 
+    <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Cras maximus feugiat elementum. Pellentesque aliquet in lacus laoreet hendrerit. Proin eget tellus odio. Sed posuere sem eget blandit cursus. Curabitur venenatis, leo a malesuada sollicitudin, nunc diam luctus mauris, id semper urna massa maximus orci. </p>
+     <button class="adopt-button"><p>Adopt</p></button>
+        </section>
+</div>
+     <footer>
+    <div class="footer-holder">
+    <div class="footer-img-holder"> <img src="images/adoptadog-logo.svg">
+        </div>   
+    <div class="footer-text-holder">
+    <p class="footer-text">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec vulputate ante ac tellus sagittis pretium. Sed at nulla justo. Nam sed tincidunt nibh, at sodales ex. </p>
+    </div>
+    <nav class="footer-nav">
+    <ul>
+        <li><a href="#home">Home</a></li>
+        <li>/</li>
+        <li><a href="#dogs">Dogs</a></li>
+        <li>/</li>
+        <li><a href="#contact">Contact</a></li>
+        
+        </ul>    
+     </nav>
+    <p>&#169; 2018 Adoptadog, LLC | Seattle, WA</p>
```

Comment:
you can also use `copy;` to show the copy right symbol. It's better to use that syntax in your HTML